src/intro-to-c/random/random-intro.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Introduction to Random Number Generation
+author: Johnson Laguerre
+---
+
+# Introduction to Random Number Generation.
```

Comment:
Generally since the page already has a title, you may want to omit the first top-level heading.

![image](https://user-images.githubusercontent.com/7614459/225792985-22e35eae-c6f9-45d2-a4d6-7fe1605aae4b.png)